app.js

```diff
@@ -40,7 +40,6 @@ var makeHeaderRow = function() {
   var thEl = document.createElement('th');
   thEl.textContent = ' ';
   trEl.appendChild(thEl);
-  // cookieTable.appendChild(trEl);
 
```

Comment:
Can't comment at the top right now, but you're missing 'use strict'; at the top.